bids/layout/layout.py

```diff
@@ -66,12 +52,11 @@ class BIDSLayout(object):
     root : str
         The root directory of the BIDS dataset.
     validate : bool, optional
-        If True, all files are checked for BIDS compliance
-        when first indexed, and non-compliant files are ignored. This
-        provides a convenient way to restrict file indexing to only those
-        files defined in the "core" BIDS spec, as setting validate=True
-        will lead files in supplementary folders like derivatives/, code/,
-        etc. to be ignored.
+        If True, all files are checked for BIDS compliance when first indexed,
+        and non-compliant files are ignored. This provides a convenient way to
+        restrict file indexing to only those files defined in the "core" BIDS
+        spec, as setting validate=True will lead files in supplementary folders
+        like derivatives/, code/, etc. to be ignored.
     absolute_paths : bool, optional
```

Comment:
Good idea to note that it's deprecated in the docs.

Reply:
I decided not to deprecate it (for now at least), as it turns out it's also used outside of indexing.